examples/plotting/multi_scale.py

```diff
@@ -1,3 +1,13 @@
+'''This example shows a graph with multiple scales on y-axis.
+This can be used to depict values in two different units.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.line, bokeh.plotting.figure.circle,
+    bokeh.plotting.figure.add_layout
```

Comment:
This breaks the docs build. All the APIs have to go on a single line

Reply:
Yes, the guidelines said to have a maximum of 80 characters in a line so I wrote it in the new line, will do the required changes.